src/js/components/SearchPage.js

```diff
@@ -98,14 +98,17 @@ export default function SearchPage() {
 
   const isResourceSearch = activeFacets["type"].includes(LR_TYPE_RESOURCEFILE)
 
+  const facetMap = [
+    ["topics", "Topics"],
+    ["department_name", "Department"]
+  ]
```

Comment:
for this variable here I think we probably want to move it out of the component and just declare it as a top-level variable in this module, just because we aren't mutating it or anything.